src/api/admin.py

```diff
@@ -205,11 +206,13 @@ async def delete_alcohol(
 async def update_alcohol(
         alcohol_id: str,
         payload: AlcoholUpdate,
-        db: Database = Depends(get_db)
+        db: Database = Depends(get_db),
+        settings: ApplicationSettings = Depends(get_settings)
 ):
     """
     Update alcohol by id
     """
+    old_alcohol = await AlcoholDatabaseHandler.get_alcohol_by_id(db.alcohols, alcohol_id)
```

Comment:
dopiero niżej masz walidacje czy to dobry id. Więc powinno to się znaleźć linie pod tym